projects/bp-gallery/.eslintrc

```diff
@@ -81,7 +81,8 @@
     "@typescript-eslint/no-unnecessary-type-assertion": "error",
     "@typescript-eslint/no-non-null-asserted-optional-chain": "error",
     "@typescript-eslint/prefer-optional-chain": "warn",
-    "@typescript-eslint/prefer-nullish-coalescing": "warn"
+    "@typescript-eslint/prefer-nullish-coalescing": "warn",
+    "no-underscore-dangle": 0
```

Comment:
Where do we need this?

Reply:
seems to be unnecessary now, but I wouldn't have added it if there wasn't a underscore variable throwing an eslint error somewhere